### DISCUSSION TITLE
Cannot Access My Account

### DISCUSSION THREAD

**Discussion by emekaanekwe** [community/community#123557] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

My name is Emeka Anekwe--the account holder of [redacted by moderator] and I have been locked out of it.

[redacted by Emeka Anekwe]

Regards,
Emeka

**Comment by ebndev** [comment]
Hi @emekaanekwe,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.
